resolwe_runtime_utils.py

```diff
@@ -48,51 +137,168 @@ def _get_json(value):
         return json.loads('"{}"'.format(value))
 
 
-def save(key, value):
-    """Convert the given parameters to a JSON object.
+def command(name, data):
+    """Create a command from command name and payload."""
+    return {"type": "COMMAND", "type_data": name, "data": data}
+
 
-    JSON object is of the form:
-    { key: value },
-    where value can represent any JSON object.
+def send_data(s, data, size_bytes=5):
```

Comment:
`size_bytes` implies that it should be set to the size of the `data`, not the initial message's size. 

Would it be possible to design the protocol without fix-sized messages, e.g. with some special sequence delimiting messages?